test/timing_load_creds.c

```diff
@@ -94,14 +94,23 @@ static void usage(void)
 # endif
 #endif
 
+#if !defined(RUSAGE_SELF)
```

Comment:
The other way is to apply this patch, to ensure that this inclusion works under the same conditions as the code that uses it (thereby avoiding build breakage on Windows)


```suggestion
#if !defined(RUSAGE_SELF) && defined(_POSIX_VERSION) && _POSIX_VERSION >= 200112L
```